astro-cloudinary/tsconfig.json

```diff
@@ -1,3 +1,8 @@
 {
-  "extends": "astro/tsconfigs/strict"
-}
\ No newline at end of file
+  "extends": "astro/tsconfigs/strict",
+  "compilerOptions": {
+    "noEmit": false,
+    "declaration": true,
+    "emitDeclarationOnly": true
```

Comment:
Thanks to those you'll see those errors earlier, in the IDE. Since you were disabling the declaration emit here... it had no chance to error in the IDE and it could only error when u actually tried to emit this using `tsup`

Reply:
ha thanks! ill make sure all projects have this fix as well

Reply:
note that if u are developing a deployable app you don't need this, in those contexts u don't care about dts emit so it's OK to not have those settings